py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -222,6 +295,40 @@ def native_group_norm(
     reshaped_output = impl.shuffle.reshape(
         ctx, target, source_ir, f"{name}_reshape_output", output, shape
     )
+    reshaped_gamma = impl.shuffle.reshape(
+        ctx,
+        target,
+        source_ir,
+        f"{name}_reshape_gamma",
+        weight,
+        weight_bias_shape,
+    )
+
+    reshaped_output = impl.elementwise.mul(
+        ctx,
+        target,
+        source_ir,
+        f"{name}_mul_gamma",
+        reshaped_output,
+        reshaped_gamma,
+    )
+
+    reshaped_bias = impl.shuffle.reshape(
+        ctx,
+        target,
+        source_ir,
+        f"{name}_reshape_beta",
+        bias,
+        weight_bias_shape,
+    )
+    reshaped_output = impl.elementwise.add(
+        ctx,
+        target,
+        source_ir,
+        f"{name}_add_beta",
+        reshaped_output,
+        reshaped_bias,
+    )
```

Comment:
How is the TensorRT INormalization layer different from this? I am assuming that the implementation has been changed since the INormalization layer would not work for Group norm case. Where was it going wrong?

Reply:
I don't know how TRT implementation is different, but in reality INormalization layer does not produce correct results under FP16 and stable diffusion fails on that. The break-down implementation produces correct results